library/sync/src/main/kotlin/org/cru/godtools/sync/task/FollowupSyncTasks.kt

```diff
@@ -26,7 +25,7 @@ internal class FollowupSyncTasks @Inject internal constructor(
                 .map { followup ->
                     async {
                         try {
-                            followup.languageCode?.let { followup.setLanguage(dao.find(it)) }
+                            followup.setLanguage(languagesRepository.findLanguage(followup.languageCode))
```

Comment:
It looks to me like the old code only called setLanguage if languageCode existed, but now we are calling it regardless. This is likely my ignorance of Kotlin, but, what happens if you pass in a `null` `languageCode` here? Maybe we should have a test for that case?

Reply:
it won't be null anymore because `languageCode` is now defined as non-null. The underlying model changed in other refactoring and I noticed the warning on the null-safe operator (`?.`) when working with this code.

If the model ever changes back to having `languageCode` as nullable, this code will throw a compile-time error and we will have to address the null condition before we can compile

Reply:
The referenced property:

https://github.com/CruGlobal/godtools-android/blob/develop/library/model/src/main/kotlin/org/cru/godtools/model/Followup.kt#L23